app/controllers/application_controller.rb

```diff
@@ -5,6 +5,8 @@ class ApplicationController < ActionController::Base
   include CanCan::ControllerAdditions
 
   private
+  # Interesting that you chose to use authlogic for authentication... what
+  # led you to make that decision?
```

Comment:
I began watching your CanCanCan tutorial and THEN found out that I needed authentication first; extremely cursory research led me to authlogic.